src/ssl_stubs.c

```diff
@@ -561,7 +561,13 @@ CAMLprim value ocaml_ssl_digest(value vevp, value vcert)
 
 CAMLprim value ocaml_ssl_get_client_verify_callback_ptr(value unit)
 {
+#ifdef NO_NAKED_POINTERS
```

Comment:
This will be the default in OCaml 5, right?

Reply:
yes

Reply:
https://github.com/ocaml/ocaml/blob/f3ea33a4c9b0d2872b0bcccbcce5428b2e468373/runtime/caml/m.h.in#L85

Reply:
The code here before was a bit rubbish because it was allocating a new block every time it was called - I've changed to a single static value instead.